Makefile

```diff
@@ -14,17 +14,23 @@ install: aws-vault
 	cp -a ./aws-vault $(INSTALL_DIR)
 	codesign --options runtime --timestamp --sign "$(CERT_ID)" $(INSTALL_DIR)/aws-vault || true
 
-binaries: aws-vault-linux-amd64 aws-vault-linux-arm64 aws-vault-linux-ppc64le aws-vault-linux-arm7 aws-vault-android-arm64 aws-vault-darwin-amd64 aws-vault-windows-386.exe aws-vault-freebsd-amd64
+binaries: aws-vault-linux-amd64 aws-vault-linux-arm64 aws-vault-linux-ppc64le aws-vault-linux-arm7 aws-vault-android-arm64 aws-vault-darwin-amd64 aws-vault-darwin-arm64 aws-vault-darwin-arm7 aws-vault-windows-386.exe aws-vault-freebsd-amd64
 
 clean:
 	rm -f ./aws-vault ./aws-vault-*-* ./SHA256SUMS
 
-release: binaries aws-vault-darwin-amd64.dmg SHA256SUMS
+release: binaries aws-vault-darwin-amd64.dmg aws-vault-darwin-arm64.dmg aws-vault-darwin-arm7.dmg SHA256SUMS
 	@echo "\nTo update homebrew-cask run\n\n    cask-repair -v $(shell echo $(VERSION) | sed 's/v\(.*\)/\1/') aws-vault\n"
 
 aws-vault-darwin-amd64: $(SRC)
 	GOOS=darwin GOARCH=amd64 go build $(BUILD_FLAGS) -o $@ .
 
+aws-vault-darwin-arm64: $(SRC)
+	GOOS=darwin GOARCH=arm64 go build $(BUILD_FLAGS) -o $@ .
+
+aws-vault-darwin-arm7: $(SRC)
+	GOOS=linux GOARCH=arm GOARM=7 go build $(BUILD_FLAGS) -o $@ .
```

Comment:
What device is arm7 on darwin for?

Reply:
Good point! I removed it. I think arm64 would suffice.